### DISCUSSION TITLE
License for minutes - CC-BY-ND?

### DISCUSSION THREAD

**Discussion by fkr** [SovereignCloudStack/issues#115] [General]
Ahoi,

I've been discussing with @itrich on which license to pick for our [minutes](https://github.com/sovereignCloudStack/minutes) repository.
Both of us think, that CC-BY-ND is a good choice. I'm going to commit such a LICENSE file in a few minutes, however this is still tbd.
*Please* raise your voice if you disagree. We're also going to raise this topic in tomorrows weekly community call.

Why did we choose CC-BY-ND? Both of us feel it would be awkward if remixing of contents from the minutes would be allowed, since that would rip items out of their original context. 

felix

**Comment by fkr** [comment]
@itrich and I raised this in this weeks community call on thursday. No objections came in that call.

**Reply by itrich** [reply]
I'm currently wondering how to meaningfully manage discussions. This is an "open-ended" discussion in the "General" category and thus can't be closed/answered. Shall we migrate this to an issue which we then close subsequently?

**Comment by fkr** [comment]
Thought about this. Does it hurt, if it stays like it is? I kinda see the Discussions as a forum (or message board). 🤔 
(Phrased other way around: Does it make sense, for items like this to stay around? :)

**Reply by itrich** [reply]
This may invoke the impression that this discussion hasn't been solved, but I generally agree on that this can be safely stick around. Hoped that there is a way to somehow visually *solve* this discussion.
